legend-engine-xt-sql-query/src/main/java/org/finos/legend/engine/query/sql/model/SchemaColumn.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package org.finos.legend.engine.query.sql.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class SchemaColumn
+{
+    private final String name;
+    private final String type;
+
+    public SchemaColumn(@JsonProperty("name") String name, @JsonProperty("type") String type)
+    {
+        this.name = name;
+        this.type = type;
+    }
+
+    public String getName()
+    {
+        return name;
+    }
+
+    public String getType()
+    {
+        return type;
+    }
+
+    @Override
+    public boolean equals(Object o)
```

Comment:
I will suggest to avoid using commons equals/has, and let IntelliJ generate these methods with only JDK dependencies.  

The issue with these tools is that if you end calling equals and hashcode a lot, these will perform worst that if using just JDK.  

For example, each call needs to allocate a new object to do their function, which adds up when executing these methods often.

Reply:
That makes sense. Updated in the latest revision.